### PR TITLE
Don't change mute state while in the lobby

### PR DESCRIPTION
**Pull request by SimonBrandner**
Adjust https://github.com/vector-im/element-call/pull/1297

CC @toger5 and @daniel-abramov

### CODE REVIEW

File: src/room/GroupCallView.tsx [hunk 1]
```diff
@@ -293,7 +295,7 @@ export function GroupCallView({
           setUserChoices(choices);
           enter();
         }}
-        muteAudio={participants.size > 8}
+        initWithMutedAudio={participants.size > 0}
```
Comment (toger5) [inline src/room/GroupCallView.tsx]: This should be > 8 ?
Or did the logic change?

**Review by toger5 — COMMENTED** [review]
I think there should be an 8. Maybe we also want this to become a constant somewhere?

File: src/room/GroupCallView.tsx (hunk 1 above)
Comment (SimonBrandner) [reply src/room/GroupCallView.tsx]: Oops

**Review by toger5 — APPROVED** [review]
I was just about proposing to make it a const. Great you made it a const anyways

**Pull request merged**